about.html

```diff
@@ -0,0 +1,128 @@
+          </div>
+          <div class="nav-wrap">
+            <a class="nav-link" id="about" href="index.html" >Home</a>
+            <a class="nav-link" href="#" >Program</a>
+            <a class="nav-link" href="#" >Join</a>
+            <a class="nav-link" href="#" >News</a>
+            <a class="nav-link" href="#" >Sponsor</a>
+            <a class="nav-link" id="home" href="index.html">RW Compaign</a>
+          </div>
+        </nav>
+        <div class="menu">
+          <i class="fas fa-bars"></i>
+        </div>
+      </header>
+    <div class="banner">
+        <div class="hidden_a">
+        <div class="hidden_b">          
+        <h1>"Rwanda Film Festival"</h1>
+        <h2>Visit Rwanda Festivals and Events</h2>
+        <p>Better known as ‘Hillywood’, the Rwanda Film Festival has been bringing films from around Rwanda, Africa and the world to Kigali every July for more than 10 years now</p>
+        <h3>24th SEPTEMBER 2021 SAVE THE DATE.</h3>
+        <h5>Please contact us per Email for any futher questions about Visit Rwanda 2021! <br> <b>mucyosofa@gmail.com</b></h5>
+            </div>
+         </div>
+    </div>
+
+    <section class="main">
+        <h2>Visit Rwanda Now</h2>
+        <div class="hr">
+         </div>
+         <div class="global">
+           <h3>Rwanda is now open. Learn more here about Virunga Nation Park.</h3>  
+         </div>
+
+         <div class="imagelogo"><img src="images/gorilla-logo.png" alt="logo"></div>
+
+         <div class="lineb"></div>
+         <div class="rda">
+             <h2>Rwanda lies just south of the Equator</h2></div> 
+             <div class="hr"></div>
+             
+         <div class="view">
+             <h6>View special offers from Rwandan tour and travel operators here.</h6>
+         </div>
+         <div class="dance">
+            <h1>2021 <br> Tradition Dance</h1> 
+            
+           </div>
+         <div class="gorila">
+             <h3>2020 <br> Nation Park</h3>  
+         </div>
+        </section>
+    <footer>
+      <div class="partner" id="sponsor">
+        <h2>Partner</h2>
+    
+        <div id="sponsors">
+        <div id="sponsor_1">mozilla</div>
+        <div id="sponsor_2">Google</div>
+        <div id="sponsor_3">NAVER</div>
+        <div id="sponsor_4">daumkakao</div>
+        <div id="sponsor_5">NAVER</div>
+    </div></div>
+        <div class="address">
+            <div class="creative"><img src="images/gorilla-logo.png" alt="image">
+            </div>
+            <div class="footer_text">
+                <h3>2021 Creative Commons Rwanda. Some Rights Reserved.</h3>
+                <p>
+                    His Excellency Paul Kagame, President of Rwanda, and Commonwealth Secretary-General, The Rt. <br> Hon. Patricia Scotland QC, are today announcing the postponement of CHOGM 2021 as a result of <br> the continuing impact of the Covid-19 pandemic.  
+                </p>
+
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] I can some white space on your footer whereas the footer is supposed to occupy all the space. Please kindly fix that.
- [ ] I have also noticed that your logo in your footer is too stretched. I suggest that you should modify the width and height of the logo.

Reply:
- [ ] As said previously, there is a white space on the footer of the About page. Please kindly fix that.
- [ ] Please try and adjust the gorilla logo on the footer so that it stays on the same line with the other text on the right